### PR TITLE
Blog/embertimes 184

### PR DESCRIPTION
**Pull request by fozy81**
## Ember Times review template

- [x] Add your name to top and bottom
- [x] Put emoji in writeup title
- [x] Add a short blurb (could be the title) to the beginning
- [x] Link to external article/repo/etc in paragraph/body text, not just the writeup title (link in writeup title is optional now)
- [x] Add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account
- [x] Check that all links work

## What it does
Ember Berlin 4 videos
de the issue(s) to which this pull request relates to or which issue it closes. E.g. "Closes #1234" -->

### CODE REVIEW

File: content/the-ember-times-issue-184.md [hunk 1]
```diff
@@ -14,6 +15,7 @@ tags:
 👋 Emberistas! 🐹
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Introducing the a11y-disabled-modifier Modifier 💚, Emberliners Talks 📽,
 The Road to Ember 4.0 Polaris 🛣,
 Introducing the a11y-disabled-modifier Modifier 💚,
```
Comment (jayjayjpg) [inline content/the-ember-times-issue-184.md]: I just noticed that if we rearranged these list items as follows, it'd also match the order of articles in the post - what do you think?

```suggestion
The Road to Ember 4.0 Polaris 🛣,
Introducing the a11y-disabled-modifier Modifier 💚,
Emberliners Talks 📽,
```

File: content/the-ember-times-issue-184.md [hunk 2]
```diff
@@ -56,6 +58,22 @@ Try it out today by running `ember install a11y-disabled-modifier` on your Ember
 
 ---
 
+## [Ember Berlin Meetup with 4 talks 📽](https://www.youtube.com/watch?v=c-sWTZJ6nlM&ab_channel=Pusherl)
+
+Four amazing talks from our friends at [Emberliners meetups](https://www.meetup.com/Ember-js-Berlin/) now available for binge watching!
```
Comment (jayjayjpg) [inline content/the-ember-times-issue-184.md]: I think it'd be

```suggestion
Four amazing talks from our friends at [Emberliners meetups](https://www.meetup.com/Ember-js-Berlin/) are now available for binge watching!
```

What are your thoughts?

**Review by jayjayjpg — COMMENTED** [review]
This reads great! I left a few more typo suggestions, but otherwise this is good to go! :sparkles:

**Comment by fozy81** [comment]
@jayjayjpg Thanks for the feedback. All set to roll!

**Comment by amyrlam** [comment]
thank you both, awesome writeup! i went ahead and made a commit to add the github handle, we've been doing `First Last (@github-handle)` ... not sure if it's the best, it's sort of because everyone is on github, may not be on twitter / discord / other. i'll go ahead and merge

fozy81 - i'll get you sorted on github also

**Pull request merged**